packages/workbox-broadcast-cache-update/BroadcastCacheUpdate.mjs

```diff
@@ -61,8 +61,9 @@ class BroadcastCacheUpdate {
   }
 
   /**
-   * @return {BroadcastChannel} The BroadcastChannel instance used for
-   * broadcasting updates.
+   * @return {BroadcastChannel|undefined} The BroadcastChannel instance used for
+   * broadcasting updates, or undefined if the browser doesn't support the
```

Comment:
This doesn't actually make sense - looking at the getChannel code - it's always creating a broadcast channel - so it never returns undefined.

Reply:
Uch, I was locally running some test scenarios and I undid the line of code that was the whole point of this PR, and then submitted it.